### PR TITLE
Run/Execute command fix.

### PR DESCRIPTION
**Pull request by zhuang-weiming**
## Motivation and Context

Fixes #
## Checklist:

- [] I have added a [Signed-off-by](https://github.com/hyperledger/fabric/blob/master/CONTRIBUTING.md#legal-stuff).
- [] I have either added documentation to cover my changes or this change requires no new documentation.
- [] I have either added unit tests to cover my changes or this change requires no new tests.
- [] I have run [golint](https://github.com/golang/lint) and have fixed valid warnings in code I have added or modified. This tool generates false positives so you may choose to ignore some warnings. The goal is clean, consistent, and readable code.

The continuous integration build process will run [make checks](https://github.com/hyperledger/fabric/blob/master/Makefile#L22) to confirm that tests pass and that code quality meets minimum standards. You may optionally run this locally as PRs will not be accepted until they pass.

Signed-off-by:

### CODE REVIEW

**Comment by zhuang-weiming** [comment]
update the Run/Execute command in page of 
[https://github.com/hyperledger/fabric/blob/master/docs/dev-setup/install.md](https://github.com/hyperledger/fabric/blob/master/docs/dev-setup/install.md)
When I install hyperledger, I do not know where is my "peer"....
Please kindly check and review.

**Comment by srderson** [comment]
This should not be necessary. If you build the peer with `make peer`, it places `peer` executable in `/hyperledger/build/bin/` which should be on your PATH. What command are you using to build the peer?

File: docs/dev-setup/install.md [hunk 1]
```diff
@@ -30,7 +30,8 @@ make peer
 
 To see what commands are available, simply execute the following commands:
 ```
-$ peer
+cd $GOPATH/src/github.com/hyperledger/fabric/build/bin
+./peer
 ```
```
Comment (ghaskins) [inline docs/dev-setup/install.md:35]: This document was written specifically for vagrant.  In that context, $GOPATH/src/github.com/hyperledger/fabric/build/bin is already in the users path (indirectly, via /hyperledger mountpoint) and therefore "peer" alone should be correct.

File: docs/dev-setup/install.md [hunk 2]
```diff
@@ -48,6 +49,7 @@ You should see some output similar to below (**NOTE**: The root command below is
     Flags:
       -h, --help[=false]: help for peer
           --logging-level="": Default logging level and overrides, see core.yaml for full syntax
+          --test.coverprofile="coverage.cov": Done
```
Comment (ghaskins) [inline docs/dev-setup/install.md:52]: This seems like an inadvertent hunk.  Did you mean to include this?

File: docs/dev-setup/install.md (hunk 1 above)
Comment (ghaskins) [inline docs/dev-setup/install.md:35]: I should add that I have no problem with a note that mentions where the output goes.

**Comment by zhuang-weiming** [comment]
1. After intallation of hyperledger, my $PATH is "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/opt/go/bin:/opt/gopath/bin:/hyperledger/devenv/tools:/hyperledger/build/bin", almsot users can not know where is the peer file.
   By the way, /hyperledger/build/bin is not exist, is there anything wrong about my installation?
2. about the "--test.coverprofile="coverage.cov": Done", it is shown under command "./peer"

```
vagrant@ubuntu-1404:/opt/gopath/src/github.com/hyperledger/fabric/build/bin$ ./peer
2016/06/29 01:42:59 Load docker HostConfig: &{Binds:[] CapAdd:[] CapDrop:[] ContainerIDFile: LxcConf:[] Privileged:false PortBindings:map[] Links:[] PublishAllPorts:false DNS:[] DNSSearch:[] ExtraHosts:[] VolumesFrom:[] NetworkMode:host IpcMode: PidMode: UTSMode: RestartPolicy:{Name: MaximumRetryCount:0} Devices:[] LogConfig:{Type: Config:map[]} ReadonlyRootfs:false SecurityOpt:[] CgroupParent: Memory:0 MemorySwap:0 MemorySwappiness:0 OOMKillDisable:false CPUShares:0 CPUSet: CPUSetCPUs: CPUSetMEMs: CPUQuota:0 CPUPeriod:0 BlkioWeight:0 Ulimits:[]}
01:42:59.959 [crypto] main -> INFO 002 Log level recognized 'info', set to INFO


Usage: 
  peer [command]

Available Commands: 
  node        node specific commands.
  network     network specific commands.
  chaincode   chaincode specific commands.
  help        Help about any command

Flags:
  -h, --help[=false]: help for peer
      --logging-level="": Default logging level and overrides, see core.yaml for full syntax
      --test.coverprofile="coverage.cov": Done


Use "peer [command] --help" for more information about a command.
```

So maybe it should show the fact display to users in guidance wiki.

**Comment by christo4ferris** [comment]
@WillZhuang you need to edit the PR description to add your DCO signoff per the project's [legal requirements ](https://github.com/hyperledger/fabric/blob/master/CONTRIBUTING.md#legal-stuff)

**Comment by ghaskins** [comment]
Hi @WillZhuang,

The fact that your prompt says "vagrant@ubuntu-1404" indicates that your vagrant environment did not fully provision.  The prompt gets updated to something like "vagrant@hyperledger-devenv:0.0.10..." as the last step of provisioning.  Therefore, when we see something like yours, it's an indication that something failed during the setup.  A few comments about that:

1) We need to do a better job of informing the user that their system is likely broken. I am thinking a MOTD on login that clearly gets the users attention.
2) Users in China often run into this because the provisioning process needs to access some URLs that seem to be blocked.

As far as this patch is concerned, the instructions you have provided are not technically correct.  However, that said, the point is taken on user clarity.  Therefore, while I have to respectfully NACK this change, I would be amenable to a note of some kind that mentions where the binary ends up somewhere in the vicinity.  Would you be willing to refactor your patch in this fashion and resubmit?

**Comment by zhuang-weiming** [comment]
> The fact that your prompt says "vagrant@ubuntu-1404" indicates that your vagrant environment did not fully provision. The prompt gets updated to something like "vagrant@hyperledger-devenv:0.0.10..." as the last step of provisioning. 

OK. This is really help me a lot. 
I am not in the center of correct road, so what I applied here is not what should happen correctly. please kindly close this request by your way.

I would re-install hyperledger again "vagrant destroy","vagrant up", or any other method ?

**Comment by ghaskins** [comment]
Closing per agreement

**Pull request closed without merging**